ihor.bohdan/second/task 1/tamagochi.rb

```diff
@@ -0,0 +1,88 @@
+class Tamagochi
+  attr_accessor :health
+
+  NAMES = %w[rules state play feed hydrate].freeze
```

Comment:
name of constant should describe what's inside, maybe we should rename it? Thoughts?

Reply:
done